### DISCUSSION TITLE
How use Input useInput with many fields in Next-UI?

### DISCUSSION THREAD

**Discussion by racielfg01** [heroui-inc/heroui#856] [Feedback]
How use Input useInput with many fields? Thanks

**Comment by Xariwey** [comment]
try this,

keep in mind input1..  or input2... are just suggested names it would be best to use names that makes sense to your code
ex, name..  .addres...  phone...   etc etc..

```js
const {
	value: input1Value,
	reset: input1Reset,
	bindings: input1Bindings,
} = useInput('')

const {
	value: input2Value,
	reset: input2Reset,
	bindings: input2Bindings,
} = useInput('')
```
